src/server/bg_services/tier_ttf_worker.js

```diff
@@ -139,8 +166,8 @@ class TieringTTFWorker {
         return bucket.tiering.tiers.find(t => String(t.tier._id) === String(tier._id)).order;
     }
 
-    async _build_chunks(chunk_ids, next_tier) {
-        return this.client.scrubber.build_chunks({ chunk_ids, tier: next_tier }, {
+    async _build_chunks(chunk_ids, next_tier, cache_evict) {
+        return this.client.scrubber.build_chunks({ chunk_ids, tier: next_tier, cache_evict }, {
```

Comment:
You called this just `evict` in scrubber_api so that should fail you on schema validation.
```suggestion
        return this.client.scrubber.build_chunks({ chunk_ids, tier: next_tier, evict: cache_evict }, {
```